lib/svg-sprite/shape.js

```diff
@@ -835,10 +835,10 @@ SVGShape.prototype._replaceIdAndClassnameReferencesInCssSelectors = function(str
             const csssel = new CssSelectorParser();
             let sel = csssel.parse(selText);
             const ids = [];
```

Comment:
Not related to this PR, I just saw it here, we could probably use a Set for `ids` and possibly in other places where we expect unique data.